src/server/schema/user.schema.ts

```diff
@@ -166,3 +166,8 @@ export type CompleteOnboardingStepInput = z.infer<typeof completeOnboardStepSche
 export const completeOnboardStepSchema = z.object({
   step: z.nativeEnum(OnboardingStep).optional(),
 });
+
+export type UserSettingsSchema = z.infer<typeof userSettingsSchema>;
+export const userSettingsSchema = z.object({
+  showNewsletterDialogAt: z.date().nullish(),
```

Comment:
This var name is confusing - I'd rather `newsLetterDialogLastSeenAt` or smt like that. The wording makes it seem we'll show it at that date specifically